travis/benchmarks.lst

```diff
@@ -1,11 +1,12 @@
-micro.sort
 ml.bayes
+ml.kmeans
+ml.lr
 ml.pca
 ml.gbt
 ml.rf
 ml.svd
+ml.linear
 ml.lda
 ml.svm
-websearch.nutchindexing
-sql.scan
+
```

Comment:
Let's keep the original benchmark lists, due to it represents almost all submodules(sql, micro, etc...)

Reply:
@gczsjdy  The original benchmark lists have not included all submodules ,  you can  refer to the benchmarks.lst under this path(HiBench/conf/benchmarks.lst) that it included all submoudles.

Reply:
At least it includes micro, sql and websearch, I mean you cannot remove them...

Reply:
@gczsjdy  I haven't remove them and just split it into two for testing.